src/main/java/io/jenkins/plugins/artifact_manager_s3/S3BlobStore.java

```diff
@@ -199,14 +211,28 @@ public URL toExternalURL(@NonNull Blob blob, @NonNull HttpMethod httpMethod) thr
         return builder.build().generatePresignedUrl(container, name, expiration, awsMethod);
     }
 
+    public boolean isConfigured(){
+        return StringUtils.isNotBlank(getContainer()) && StringUtils.isNotBlank(getPrefix());
+    }
+
     @Extension
-    public static final class DescriptorImpl extends BlobStoreProviderDescriptor {
+    public static final class S3BlodStoreDescriptor extends BlobStoreProviderDescriptor {
```

Comment:
Typo, and anyway why the rename? `DescriptorImpl` is the conventional name.

Reply:
letfover of many changes